.github/workflows/ci.yml

```diff
@@ -64,6 +65,68 @@ jobs:
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
 
+  java-checks:
+    if: needs.detect-changes.outputs.java-unit-tests == 'true'
```

Comment:
Probably should consider renaming this variable eventually (not necessarily in this PR) to not confuse readers, it is more generically used now

Reply:
I aggree. I did this in the other PR.

Reply:
> the other PR
Oh, which did I miss? 🙈

Reply:
:) https://github.com/camunda/camunda/pull/19785/files#diff-1951b8794b638a238e5b4db8432cc73261ef7a353ff0a17ca3efc5501cb1dadfL14